addons/storyshots/src/test-bodies.js

```diff
@@ -41,14 +41,14 @@ export function shallowSnapshot({ story, context, renderShallowTree, options = {
   expect(result).toMatchSnapshot();
 }
 
-export function renderOnly({ story, context, renderTree }) {
-  const result = renderTree(story, context, {});
+export const renderOnly = options => ({ story, context, renderTree }) => {
+  const result = renderTree(story, context, options);
```

Comment:
Please make the "options" param to be optional. Passing an empty object is kinda confusing.

Reply:
I see. I was trying to mimic snapshotWithOptions where options are required.

I will add another export, perhaps named `renderWithOptions` thus people using `renderOnly` will not have their code broken

Reply:
I am ok with breaking change. The next release will be breaking. Or you wanted to have it in 3.4.x?

Reply:
I'm ok with either! Whichever you think is best. 

But since I think the use case for `renderOnly` with options is small so adding another export is probably best